### PR TITLE
Add a badge to the docs on README.md

### PR DESCRIPTION
**Pull request by Zeitsperre**
TEDLT

### CODE REVIEW

**Comment by tlvu** [comment]
What !!!  I am not even aware this repo is connected to RtD !

**Comment by tlvu** [comment]
Wait a sec, there is no `.readthedocs.yml` here so where is that doc coming from?  Don't think it's from this repo.

**Comment by Zeitsperre** [comment]
Wait, what?! Hold up!

**Review by huard — APPROVED** [review]
Please add birdhouse/README.md to the doc build.

**Review by tlvu — APPROVED** [review]
LGTM

File: docs/source/deployment.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Deployment
+==========
+
+.. include:: ../../birdhouse/README.rst
```
Comment (tlvu) [inline docs/source/deployment.rst:4]: Oh neat, that's how Sphinx can refer to files outside of the `docs` folder?  Awesome, I didn't know this !  

With this trick we can get the best of both worlds: in place docs when browsing on github and dedicated RtD site.

**Comment by Zeitsperre** [comment]
@huard It's a bit threadbare but good enoguh for now?

File: birdhouse/README.rst [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+* 
+  User running ``pavics-compose.sh`` below must not be ``root`` but a regular user
+  belonging to the ``docker`` group.
+
+* 
+  Install latest docker-ce and docker-compose for the chosen distro (not the
+  version from the distro).
+
+To run ``docker-compose`` for PAVICS, the `\ ``pavics-compose.sh`` <pavics-compose.sh>`_ wrapper script must be used.
```
Comment (tlvu) [inline birdhouse/README.rst:26]: Links to `pavics-compose.sh`, `env.local.example`, and `scripts/create-wps-pgsql-databases.sh`, `scripts/create-magpie-users`, `scripts/bootstrap-instance-for-testsuite`, `vagrant_variables.yml.example`, `vagrant-utils/disk-resize` broken when browsing on github.

File: birdhouse/README.rst [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+Manual steps post deployment
+----------------------------
+
+Change geoserver default admin password
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+* 
+  Go to
+  https://\ :raw-html-m2r:`<PAVICS_FQDN>`\ /geoserver/web/wicket/bookmarkable/org.geoserver.security.web.UserGroupRoleServicesPage (Security -> Users, Groups, and Roles)
```
Comment (tlvu) [inline birdhouse/README.rst:89]: Sample link appear funny with tripple '/' when browsing on github.

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        :target: https://birdhouse-deploy.readthedocs.io/en/latest/?badge=latest
+        :alt: Documentation Status
+
+PAVICS
+------
+
+Power Analytics and Visualization for Climate Science - Powered by Birdhouse and other ESGF software
+
+How to deploy the entire PAVICS platform, see the
+`README <birdhouse/README.md>`_ and the various extra components
```
Comment (tlvu) [inline README.rst:18]: This is `.rst` now instead of `.md`.

File: birdhouse/README.rst [hunk 5]
```diff
@@ -0,0 +1,259 @@
+.. role:: raw-html-m2r(raw)
```
Comment (tlvu) [inline birdhouse/README.rst:1]: Just curious, what is this `role:: raw-html-m2r(raw)`?

**Review by tlvu — CHANGES_REQUESTED** [review]
Sorry just caught some last minute problems when browsing on github.  Maybe it's github `.rst` rendering specific problem?

**Comment by tlvu** [comment]
On https://birdhouse-deploy.readthedocs.io/en/latest/deployment.html all the links are broken because they are relative to the layout on github :(

File: birdhouse/README.rst (hunk 5 above)
Comment (Zeitsperre) [reply birdhouse/README.rst:1]: `m2r` is a library for converting markdown to rst. If we need to push some fixes, I'll address this.

**Pull request merged**